lib/namespacing-transform.js

```diff
@@ -1,5 +1,23 @@
 'use strict';
 
+const TRANSLATE_HELPER = 'ember-holy-futuristic-template-namespacing-batman@-translate-dynamic';
+
+function rewriteOrWrapComponentParam(node, b) {
+  if (!node.params.length) {
+    return;
+  }
+  let firstParam = node.params[0];
+  if (firstParam.type === 'StringLiteral') {
+    // handle string constant
+    node.params[0] = b.literal('StringLiteral', firstParam.original.replace('::', '@'), firstParam.loc);
```

Comment:
I think we actually should use `b.string` here?

Not sure, but on Ember 2.12 `b.literal` doesn’t seem to exist and CI is failing with:

```
b.literal is not a function
```